thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/model/merge/AverageAnomalyMergeModel.java

```diff
@@ -28,18 +28,26 @@ public void update(AnomalyDetectionContext anomalyDetectionContext,
 
     double weight = 0d;
     double score = 0d;
+    double baseline = 0d;
+    double current = 0d;
     for (RawAnomalyResultDTO rawAnomaly : rawAnomalyResultDTOs) {
       weight += rawAnomaly.getWeight();
       score += rawAnomaly.getScore();
+      current += rawAnomaly.getAvgCurrentVal();
+      baseline += rawAnomaly.getAvgBaselineVal();
     }
     if (rawAnomalyResultDTOs.size() != 0) {
       double size = rawAnomalyResultDTOs.size();
       weight /= size;
       score /= size;
+      current /= size;
```

Comment:
The assumption for this model is that the length of raw anomalies are the same, e.g., the anomalies generated by splitline regression, and does a simple average across all raw anomalies.

If the length of the raw anomalies are different, this model provides an estimation. In this case, the developer should choose a different merge model.